.github/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Summary
+
+> What is the reason for this Pull-Request? If adding new functionality, please comment:
+> a code snippet showcasing the end-to-end usage of your new methods, and the exact output
+> from this code (print out any returned variables).
+
+
+
+## Impact
+
+> What is the scope of your changes and who/where do you think they will affect?
+> If your change modifies documentation or a tutorial notebook, please include
+> screenshots showing the rendered outputs for a quicker initial review.
+
+
+## Testing
+
+> What testing have you done and verified? Please link to the one unit test
+> that is most end-to-end check of your new functionality. It's ok if your
+> unit tests are not yet comprehensive when you first open the PR,
+> we can revisit them later!
+
+-
+
+## Ticket(s) or Conversations
```

Comment:
We utilize GitHub issues to track, discuss, and manage ongoing development tasks, rather than using the term "tickets".

```suggestion
## Links to Relevant Issues or Conversations
```